assignments/arrays.js

```diff
@@ -63,27 +63,53 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(inventory[32]);
```

Comment:
looks like you fixed everything up! remember what this is asking you to log
should be:
```
console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model} ` );
```
${} is a template literal, and allows you to insert variables into a string

Reply:
thanks i didn't understand that syntax at first, i just implemented it and updated everything for the MVP.